tests/unit/converters/test_pandapower_converter_input.py

```diff
@@ -1462,14 +1469,19 @@ def test_create_pgm_input_three_winding_transformers__tap_dependent_impedance()
     pp_net: pp.pandapowerNet = pp.create_empty_network()
     pp.create_bus(net=pp_net, vn_kv=0.0)
     args = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
-    pp.create_transformer3w_from_parameters(pp_net, *args, tap_dependent_impedance=True)
 
-    converter = PandaPowerConverter()
-    converter.pp_input_data = {k: v for k, v in pp_net.items() if isinstance(v, pd.DataFrame)}
+    if version.Version(metadata.version("pandapower")) >= version.Version("3"):
+        with pytest.deprecated_call():
+            pp.create_transformer3w_from_parameters(pp_net, *args, tap_dependent_impedance=True)
+    else:
+        pp.create_transformer3w_from_parameters(pp_net, *args, tap_dependent_impedance=True)
 
-    # Act/Assert
-    with pytest.raises(RuntimeError, match="not supported"):
-        converter._create_pgm_input_three_winding_transformers()
+        converter = PandaPowerConverter()
+        converter.pp_input_data = {k: v for k, v in pp_net.items() if isinstance(v, pd.DataFrame)}
+
+        # Act/Assert
+        with pytest.raises(RuntimeError, match="not supported"):
+            converter._create_pgm_input_three_winding_transformers()
 
```

Comment:
Nitpick:
```suggestion
        with pytest.raises(RuntimeError, match=r"not supported"):
```

Probably at other places as well.

Reply:
Github messed up... I only meant to comment about the raw string part...